elastic/kibana
# fix: Make sure 404 step screenshots are rendered as "Not Available" instead of indefinite loading

Fixes #142267

## Summary

Fixes the issue where browser journey step screenshots keep loading indefinitely, for browser's with failed steps or browser monitors with screenshots turned off.

### How to test
1. Set up a cluster so that you can create and run a browser monitor in Kibana. Or use the [deployment](https://p.elstc.co/paste/ux9LDNaO#r+eKEZdF-rSh6i4A0AHS0Cj7Wyg6ORELoMKsujoiSM3) if it is still live.
2. Goto Uptime -> Monitor Management
3. Create a browser monitor with at least one step and in the Advanced Settings, make sure to turn off the Screenshots option.
4. Save the monitor.
5. Wait for the results to get ingested and in Uptime -> Overview page, find the monitor record and click it.
6. You should see "Image not available" as a placeholder for step screenshots.

![image](https://user-images.githubusercontent.com/2748376/193280284-857164e4-5302-44ad-a0c9-7d62919faae2.png)
